app/config/config_prod.yml

```diff
@@ -2,6 +2,10 @@ imports:
     - { resource: config.yml }
     - { resource: services/sentry.xml }
 
+#services:
+#    app.geocoder.cache_adapter:
+#        class: 'Doctrine\Common\Cache\RedisCache'
```

Comment:
I think we should use the 3.2 Cache component, and I don't like commiting commented code.